dali/pipeline/basic/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get all the source files and dump test files
+file(GLOB tmp *.cc *.cu)
+set(DALI_SRCS ${DALI_SRCS} ${tmp})
+file(GLOB tmp *_test.cc)
```

Comment:
As long as it's WIP it doesn't really matter, but cmake discourages using `file(GLOB)` in this use case:
https://cmake.org/cmake/help/v3.3/command/file.html

> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.

Reply:
I would keep it consistent with the rest of code. We can refactor it latter in other PR.